sklearn/discriminant_analysis.py

```diff
@@ -623,11 +623,15 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
         Discriminant Analysis
     """
 
-    def __init__(self, priors=None, reg_param=0., store_covariances=False,
-                 tol=1.0e-4):
+    def __init__(self, priors=None, reg_param=0.,store_covariance=False, tol=1.0e-4,store_covariances=None):
+        if store_covariances is not None:
+        	warnings.warn("'store_covariances' was renamed to 'store_covariance' in version 0.17 and "
+                      "will be removed in later versions.", DeprecationWarning)
+        	self.store_covariance = store_covariances
+
         self.priors = np.asarray(priors) if priors is not None else None
         self.reg_param = reg_param
-        self.store_covariances = store_covariances
+        #self.store_covariances = store_covariances
```

Comment:
This is where you store the value of `store_covariance`. You need to change this commented line to:
```
self.store_covariance = store_covariance
```